ui/src/main/webapp/index.html

```diff
@@ -21,7 +21,7 @@
         <meta charset=utf-8/>
         <meta name="viewport"
             content="user-scalable=no, initial-scale=1.0, minimum-scale=1.0, maximum-scale=1.0">
-        <title>Gaffer UI</title>
+        <title>Gaffer</title>
```

Comment:
Can we just remove this line?

Reply:
I think it would be useful to have a page title set when the page first loads before javascript kicks in and loads the config and sets it. It is also useful as metadata about the webapp that is used by some tools.